README.md

```diff
@@ -69,6 +69,39 @@ For detailed installation instructions, see [Getting Started for UI](https://www
 
 Note: this web client only works on Chrome browser.
 
+## Developing with MindMeld
+
+We welcome every contribution to MindMeld platform!
```

Comment:
We are very happy to receive and merge your contributions. You can start off with looking at the open issues here: https://github.com/cisco/mindmeld/issues. You can also create your own issue and link it to your code fix in a pull request, which we will promptly review.